_data/rules.yml

```diff
@@ -426,6 +426,16 @@ services:
             query: 'probe_ssl_earliest_cert_expiry - time()  <= 0'
             severity: error
 
+  - name: wmi_exporter
+    exporters:
+      - name: martinlindhe/wmi_exporter
+        doc_url: https://github.com/martinlindhe/wmi_exporter
+        rules:
+          - name: Service Status
+            description: Windows Service state is not OK. This example uses IIS web server named "w3svc".
+            query: 'wmi_service_status{name="w3svc",status="ok"} != 1'
```

Comment:
I would remove the `name` filter here, to ensure we receive an alert for any service crashing. WDYT ?

Reply:
Thanks. Very good point. I will test this 1-2 weeks in our systems. I worry a little bit that we get flooded with alerts, because there are lots of "default" services on a Windows system. Will keep you updated on the outcome.

Reply:
Ok, thanks for your help!

Let me know as soon as you feel confident with this. Then I will merge ;) 👌